app/services/search/bing_search_service.rb

```diff
@@ -4,14 +4,15 @@ module Search
   class BingSearchService
     BING_SEARCH_URL = 'https://www.bing.com/search?q=%{keyword}&setlang=%{language}'
 
-    def initialize(keyword, language = 'en')
+    def initialize(keyword:, language: 'en')
       @search_url = format(BING_SEARCH_URL, keyword: keyword, language: language)
     end
 
     def search
       uri = URI.parse(@search_url)
 
-      response = Faraday.get(uri)
+      headers = {}.merge(UserAgentHeaderGenerator.call)
```

Comment:
Same here 😄 

Btw, the error should be

```
raise IcRailsHueyToby::Errors::SearchServiceError
```

instead of

```
raise IcRailsHueyToby::Errors::ClientServiceError
```

💭